WordPress/Classes/ViewRelated/Blog/QuickStartToursCollection.swift

```diff
@@ -59,8 +59,8 @@ struct QuickStartGrowToursCollection: QuickStartToursCollection {
         self.completedImageName = "wp-illustration-tasks-complete-audience"
         self.analyticsKey = "grow"
         self.tours = [
-            QuickStartShareTour(),
             QuickStartPublishTour(),
+            QuickStartShareTour(),
```

Comment:
I moved these because the previous order didn't flow well anymore.

Reply:
Sounds good! Can you also update the Share tour step 1 description target from "Sharing" to "Social"? We should have updated this when we first changed the copy, but I only noticed it now 🙇‍♀️ 

```
let step1DescriptionTarget = NSLocalizedString("Social", comment: "The menu item to select during a guided tour.")
```

<img src="https://github.com/wordpress-mobile/WordPress-iOS/assets/6711616/a19ae829-98e2-4f80-9ed0-7c1f9c82e58a" width=200>

Reply:
I also just noticed that we should also probably update the Quick Start icons to use the new assets - but feel free to address this later